<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdicionarContato</class>
 <widget class="QDialog" name="AdicionarContato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_nome_principal">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agenda</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_escreve_nome">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2_escreve_telefone">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3_escreve_email">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_nome_contato">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NOME :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_telefone_contato">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Telefone:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_email_contato">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adicionar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_volta_tela_inicial">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
